src/types.ts

```diff
@@ -87,16 +86,18 @@ export type Op = {
  */
 export type HashedOp = Op & {
   /**
-   * The hash of the operation, using all fields present in the op
+   * The hashed identifier for this operation, used to confirm that the operation
+   * was completed when it is received from Inngest.
    */
-  hash: string;
+  id: string;
 };
 
 /**
  * A helper type to represent a stack of operations that will accumulate
- * throughout a step function's run.
+ * throughout a step function's run.  This stack contains an object of
+ * op hashes to data.
  */
-export type OpStack = Record<string, Op>;
+export type OpStack = Record<string, any>;
```

Comment:
Why did this change to `any`?

Reply:
The state store only records the data within an Op, instead of the Op itself.  The rest of the data returned in the Op is used to construct an edge for our queue (eg. sleep in 10 minutes).

We don't need to return the full Op information to the SDK — only the data.  In the future, we can add Op information to the query params (?step=Step&op=$data) to indicate which parallel step to run.  This feels natural, as this is how we configure which step to run already :)

Reply:
@tonyhb Aha of course. Cool and big agree with putting the op to run in the query string! Awesome!